src/core/src/service_interfaces/StatusHandler.py

```diff
@@ -366,7 +363,6 @@ def __new_assessment_summary_json(self, assessment_packages_json, status, code):
         if self.vm_cloud_type == Constants.VMCloudType.ARC:
             substatus_message["patchAssessmentStatus"] = code
             substatus_message["patchAssessmentStatusString"] = status
-
         return substatus_message
```

Comment:
nit: The extra line break there is to signify a separation or sections within the logic in that function. If you observe, throughout the function there are line breaks separating code logic. We should follow a common standard all across, either no line breaks all throughout or yes to line breaks separating sections